doc/glossary.rst

```diff
@@ -7,6 +7,14 @@ Ceph is growing rapidly. As firms deploy Ceph, the technical terms such as
 that explain what each component does. The terms in this glossary are 
 intended to complement the existing technical terminology.
 
+- Defining some of these technical terms:
+"RADOS" : It stands for Reliable Autonomic Distributed Object Store (RADOS) which is an open source object storage service that is an important part of the Ceph distributed storage system. RADOS was created at the University of California at Santa Cruz. The Ceph open source community develops and maintains the software.
```

Comment:
@dsdeeptisharma Sorry to say, this does not speaks shortly about what RADOS actually is/What it does?
Only tells its full form/creators.
Your Thoughts?

Though RBD definition is bit more clear about what it does..